.github/workflows/new-issues.yaml

```diff
@@ -0,0 +1,16 @@
+on:
+  issues:
+    types: opened
+
+jobs:
+  assign_issue_to_project:
+    runs-on: ubuntu-latest
```

Comment:
I'm not familiar with this, but why does it have to run on ubuntu?

Reply:
It doesn't have to, but GitHub actions need you to specify a machine pool to run on...just like in ADO pipelines you need to specify which pool to run on